application/org.openjdk.jmc.flightrecorder.graphview/pom.xml

```diff
@@ -59,21 +59,21 @@
 				<version>${download.maven.plugin.version}</version>
 				<executions>
 					<execution>
-						<id>d3-v5-js</id>
+						<id>download-d3</id>
 						<phase>${download-maven-plugin.phase}</phase>
 						<goals>
 							<goal>wget</goal>
 						</goals>
 						<configuration>
-							<url>https://d3js.org/d3.v5.min.js</url>
+							<url>https://d3js.org/d3.v6.min.js</url>
```

Comment:
Maybe put this in a property

Reply:
Smallest possible changes this close to release.